scandium-core/src/main/java/org/eclipse/californium/scandium/auth/X509CertPath.java

```diff
@@ -97,4 +97,18 @@ public static X509CertPath fromBytes(final byte[] encodedPath) {
 	public String getName() {
 		return target.getSubjectX500Principal().getName();
 	}
+
+	/**
+	 * Gets this certificate path.
+	 *
+	 * @return The path.
+	 */
+	public CertPath getPath() { return path; }
```

Comment:
sure, I placed it this way following the style of `getName`, I'll change it soon, thanks.